adal/src/main/java/com/microsoft/aad/adal/AuthenticationResult.java

```diff
@@ -109,6 +112,21 @@
         mIdToken = idToken;
     }
 
+    AuthenticationResult(String accessToken, String refreshToken, Date expires, boolean isBroad,
```

Comment:
Can we update existing one? If we're going to add new constructors everytime we add new field in AuthenticationResult, it will be too much for the class.